packages/discovery-react-components/src/components/SearchFacets/components/FacetsGroups/CollapsibleFacetsGroup.tsx

```diff
@@ -18,6 +18,10 @@ interface CollapsibleFacetsGroupProps {
    * Facets configuration with fields and results counts
    */
   facets: (SelectableDynamicFacets | SelectableQueryTermAggregationResult)[];
+  /**
+   * Show matching documents count as part of label
+   */
+  showMatchingResults?: boolean;
```

Comment:
this should probably be required since we will always ensure its value from the parent component